### PR TITLE
Import the suppressions per report

### PR DESCRIPTION
**Pull request by dkrupp**
CodeChecker cmd suppress run_name -i <import_file>
will only import suppressions for the run indicated by run_name,
and not all reports.

Suppressions are imported as single report suppressions
instead of suppression rules.

This is the behaviour the user expects when run name is given
as a parameter to the command.

This patch also fixes the changeReviewStatus(report_id, status, message)
API function to change the review status of a single report only
instead of all reports with the same hash.

### CODE REVIEW

File: docs/web/user_guide.md [hunk 1]
```diff
@@ -521,7 +410,148 @@ Database management arguments.:
...
+```sh
+CodeChecker store ./my_plists -n my_project
+```
+
+#### `PRODUCT_URL` format
+
+Several subcommands, such as `store` and `cmd` need a connection specification
+on which server and for which *Product* (read more [about
+products](products.md)) an action, such as report storage or result
+retrieving, should be done.
```
Comment (bruntib) [inline docs/web/user_guide.md:526]:
```suggestion
Several sub-commands, such as `store` and `cmd` need a connection specification
that identifies a CodeChecker server and a *Product* (read more [about products](products.md).
Actions like report storage or result retrieving are performed towards this server and product.
```

File: docs/web/user_guide.md [hunk 2]
```diff
@@ -521,7 +410,148 @@ Database management arguments.:
...
+
+This is done via the `PRODUCT_URL` where indicated in the subcommand, which
+contains the server's access protocol, address, and the to-be-used product's
+unique endpoint. The format of this string is:
+`[http[s]://]host:port/ProductEndpoint`. This URL looks like a standar Web
```
Comment (bruntib) [inline docs/web/user_guide.md:531]:
```suggestion
`[http[s]://]host:port/ProductEndpoint`. This URL looks like a standard Web
```

**Review by bruntib — CHANGES_REQUESTED** [review]

File: web/server/codechecker_server/api/report_server.py [hunk 3]
```diff
@@ -1957,21 +1958,41 @@ def isReviewStatusChangeDisabled(self):
...
+                if status in ["false_positive", "intentional"]:
+                    if report.detection_status in [
+                            "unresolved", "new", "reopened"]\
+                        and report.review_status not in [
+                            "false_positive", "intentional"]:
+                        session.query(Report).filter(
+                            Report.id == report_id).update(
+                                {"fixed_at": datetime.now()})
+                else:
+                    if report.detection_status in [
```
[the remark this answers is not in the document]
Comment (dkrupp) [reply web/server/codechecker_server/api/report_server.py:1983]: fixed

File: docs/web/user_guide.md [hunk 4]
```diff
@@ -86,202 +101,36 @@ positional arguments:
...
- * In a web browser: http://localhost:8001
- * In the command line:
-    CodeChecker cmd results myproject
-
-Example scenario: Analyzing, and printing results to Terminal (no storage)
---------------------------------------------------------------------------
-In this case, no database is used, and the results are printed on the standard
-output.
-
-    CodeChecker check -b "cd ~/myproject && make"
```
[the remark this answers is not in the document]
Comment (dkrupp) [reply docs/web/user_guide.md:112]: I don't think we need this check related part in the web user guide.

**Comment by dkrupp** [comment]
I fixed all your comments. Ready to be merged.

File: docs/web/user_guide.md [hunk 5]
```diff
@@ -521,7 +410,148 @@ Database management arguments.:
...
+CodeChecker analyze ../codechecker_myProject_build.log -o ./my_plists
+```
+
+then the results of the analysis can be stored with this command:
+
+```sh
+CodeChecker store ./my_plists -n my_project
+```
+
+#### `PRODUCT_URL` format
```
[the remark this answers is not in the document]
Comment (dkrupp) [reply docs/web/user_guide.md]: fixed

**Review by bruntib — APPROVED** [review]

**Pull request merged**